templates/screen_test.html

```diff
@@ -1,27 +1,25 @@
 {% extends 'base.html' %}
-{% load staticfiles %}
+{% load static %}
 {% block style %}
 .container{
-  padding-top:120px;
+  margin-top:120px;
 }
-fieldset{
-  padding-top:40px;
-}
-button{
-  margin-bottom:50px;
+.screen-form {
```

Comment:
Can i see what it looks like. You can attach a screenshot

Reply:
![screen-form_1](https://user-images.githubusercontent.com/37906201/67488326-3d150c80-f635-11e9-86a0-ee59166af2c4.png)

![screen-form](https://user-images.githubusercontent.com/37906201/67488340-44d4b100-f635-11e9-8120-085f540fff56.png)